pom.xml

```diff
@@ -451,5 +454,22 @@ Note: The maven build currently needs to be started from the javaslang root dir
                 </pluginManagement>
             </build>
         </profile>
+        <!-- A profile for running the benchmarks -->
+        <profile>
+            <id>benchmark</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>${maven.exec.version}</version>
+                        <configuration>
+                            <classpathScope>test</classpathScope>
+                            <mainClass>javaslang.JmhRunner.main</mainClass>
```

Comment:
Hah! I'm a Maven Expert I think it has to be the class, not the method, i.e.

```xml
<mainClass>javaslang.JmhRunner</mainClass>
```

instead of

```xml
<mainClass>javaslang.JmhRunner.main</mainClass>
```

Could you please test if it is right?